timemachine/lib/potentials.py

```diff
@@ -34,6 +36,17 @@ def bound_impl(self, precision):
         return custom_ops.BoundPotential(self.unbound_impl(precision), self.params)
 
 
+class SummedPotential(CustomOpWrapper):
+    def __init__(self, potentials: List[CustomOpWrapper], params_init: List[NDArray]):
+        self._potentials = potentials
```

Comment:
Would be good to verify that the number of potentials agrees with the number of parameter sets

Reply:
Good call, added in 9d898d7 and tested in 6c9572b